src/app/operators/operators.component.ts

```diff
@@ -48,15 +65,17 @@ export class OperatorsComponent implements OnInit, AfterViewInit {
     private _breakpointObserver: BreakpointObserver,
     private _router: Router,
     private _activatedRoute: ActivatedRoute,
-    @Inject(OPERATORS_TOKEN) public operators: OperatorDoc[]
-  ) { }
+    @Inject(OPERATORS_TOKEN) public operators: OperatorDoc[],
+    private _seo: SeoService
+  ) {}
 
   ngOnInit() {
     this.groupedOperators = groupOperatorsByType(this.operators);
     this.categories = Object.keys(this.groupedOperators);
-    this._subscription = this._activatedRoute
-      .fragment
-      .subscribe(name => this.scrollToOperator(name));
+    this._subscription = this._activatedRoute.fragment.subscribe(name =>
+      this.scrollToOperator(name)
```

Comment:
@feloy - This subscription is not required anymore since we don't use fragments.